e2e/list/index.test.ts

```diff
@@ -73,8 +84,14 @@ describe('test list command', () => {
 
     expect(logs).toMatchInlineSnapshot(`
       [
+        "a.test.ts > test a",
```

Comment:
test suites should not be listed to avoid too much verbosity, we can add a cli option to include suites, eg `--includeSuite`